stdlib/2and3/wsgiref/validate.pyi

```diff
@@ -1,50 +1,53 @@
 import sys
-from typing import Any
+from typing import Any, Iterable, Iterator, Optional
+
+from wsgiref.types import WSGIApplication, InputStream, ErrorStream
 
 class WSGIWarning(Warning): ...
 
-def validator(application): ...
+def validator(application: WSGIApplication) -> WSGIApplication: ...
 
 class InputWrapper:
-    input = ...  # type: Any
-    def __init__(self, wsgi_input): ...
-    def read(self, *args): ...
+    input: InputStream
+    def __init__(self, wsgi_input: InputStream) -> None: ...
     if sys.version_info < (3,):
-        def readline(self): ...
+        def read(self, size: int = ...) -> str: ...
```

Comment:
Using `bytes` in the Python 2 code would make it clearer what the difference is between the Python 2 and 3 signature.

Reply:
Good point, changing it.